coco/check.py

```diff
@@ -342,11 +342,12 @@ def __init__(self, name, state_paths, on_failure, save_to_state, forwarder, stat
                         f"Found value '{field}' of type '{type(path).__name__}' "
                         f"in state-reply-check for /{name} (expected 'str')."
                     )
-                if not state.find_or_create(path):
+                if not state.exists(path):
```

Comment:
isn't the other case where `state_paths` is a `str` (line 331) also affected by this issue and should be changed too?

Reply:
Yeah, you're right!